src/main/java/uk/gov/hmcts/sscs/service/ccd/ReadCoreCaseDataService.java

```diff
@@ -26,7 +26,7 @@ public ReadCoreCaseDataService(CoreCaseDataService coreCaseDataService) {
     }
 
     public CaseDetails getCcdCaseDetailsByCaseId(String caseId) {
-        EventRequestData eventRequestData = coreCaseDataService.getEventRequestData("appealCreated");
+        EventRequestData eventRequestData = coreCaseDataService.getEventRequestData(null);
         String serviceAuthorization = coreCaseDataService.generateServiceAuthorization();
```

Comment:
Why null parameter in the request?

Reply:
`appealCreated` is never used. It should have been removed a while ago so I was just cleaning this up.